### ISSUE TITLE
About Loss Function

### ISSUE THREAD

**Issue by AIyumeng** [dongyangli-del/EEG_Image_decode#44]
In [paper](https://arxiv.org/pdf/2403.07721) Section 2.4

$$Loss = \lambda \cdot L_{CLIP} + (1 - \lambda) \cdot L_{MSE}$$

In `Retrieval/ATMS_retrieval.py`

```python
loss = alpha * img_loss + (1 - alpha) * text_loss
```

In `Generation/ATMS_reconstruction.py`

```python
loss = (alpha * regress_loss * 10 + (1 - alpha) * img_loss * 10)
```
I noticed that in Retrieval/ATMS_retrieval.py the loss function doesn’t include the MSE term as mentioned in Section 2.4.
Is there any reason for not applying MSELoss here?
